apps/Gruntfile.js

```diff
@@ -568,6 +564,17 @@ describe('entry tests', () => {
     }
   };
 
+  config.clean = {
+    all: ['build'],
+    // The karma-webpack-backed unit tests generate several hundred megabytes
+    // worth of assets in /tmp/ on each run which will accumulate indefinitely
+    // on our persistent test server unless we clean them up.
+    unitTest: {
+      options: {force: true},
+      src: [webpackOutputBasePath + '*']
```

Comment:
Note that this will remove all previously-generated directories as well, not just the one from the current run. I believe this is desirable because Grunt doesn't execute later tasks in a series if an earlier one failed, so otherwise test runs with a unit test failure will never get cleaned up. This way, they'll get cleaned up on the next successful test run.